server/data/nomisMappingClient.ts

```diff
@@ -0,0 +1,12 @@
+import logger from '../../log'
+import type { RestClient } from './restClient'
+import type { LocationMapping } from './nomisMappingClientTypes'
+
+export default class NomisMappingClient {
+  constructor(private restClient: RestClient) {}
+
+  async getDpsLocationMappingUsingNomisLocationId(nomisLocationId: number): Promise<LocationMapping> {
```

Comment:
getting the location's id from the locationsInsidePrisonApi but using the Nomis version of the id